helper/carbonapipb/carbonapi.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package carbonapipb;
+
+
+message AccessLogDetails {
+string handler = 1;
+string carbonapi_uuid = 2;
+string username = 3;
+string url = 4;
+string peer_ip = 5;
+string peer_port = 6;
+string host = 7;
+string referer = 8;
+string format = 9;
+bool use_cache = 10;
+repeated string targets = 11;
+int32 cache_timeout = 12;
+repeated string metrics = 13;
+bool have_non_fatal_errors = 14;
+string runtime = 15;
```

Comment:
It should be either int64 and unix timestamp (or unixts with nanoseconds) or google.protobuf.Timestamp (https://godoc.org/github.com/golang/protobuf/ptypes#TimestampProto)